app/models/job_type.rb

```diff
@@ -3,4 +3,13 @@
 class JobType < ApplicationRecord
   belongs_to :user
   has_many :jobs
+
+  before_save :nullify_blanks
+
+  private
+
+  def nullify_blanks
```

Comment:
I would change this to 
```ruby
def nullify_blanks
  self.environment_variables ||= "{}"
  self.working_directory ||= nil
end
```

Reply:
Does a blank string evaluate to false in a boolean context though?

Reply:
```
18:44 $ irb
2.4.1 :001 > !!""
 => true
```

Reply:
Ahh, good call.